inc/customizer/colors.php

```diff
@@ -0,0 +1,457 @@
+<?php
+
+class Primer_Customizer_Colors {
+
+	/**
+	 * Array of customizable colors.
+	 *
+	 * @var array
+	 */
+	public $colors = array();
+
+	/**
+	 * Array of available color schemes.
+	 *
+	 * @var array
+	 */
+	public $color_schemes = array();
```

Comment:
@jonathanbardo Maybe the visibility on these should be `protected` and we expose a magic getter method? This would make the filter the only way the values can be altered outside of the class.